Sources/MongoSwift/ConnectionString.swift

```diff
@@ -242,38 +242,47 @@ internal class ConnectionString {
 
     /// Sets and validates authentication-related on the underlying `mongoc_uri_t`.
     private func applyAndValidateAuthOptions(_ options: MongoClientOptions?) throws {
-        guard let credential = options?.credential else {
```

Comment:
this file is mostly whitespace changes due to some indenting, so suggest the "hide whitespace changes" option to cut down on the diff